xcell/mappers/mapper_P18SMICA_NOSZ.py

```diff
@@ -0,0 +1,44 @@
+from .mapper_base_Planck import MapperBasePlanck
+import healpy as hp
+import numpy as np
+
+
+class MapperP18SMICA_NOSZ(MapperBasePlanck):
```

Comment:
Is this specific of the NOSZ version of SMICA?

Reply:
I have only tried with SMICA-NOSZ but it should also work for SMICA since both have the same structure in how they store their half mision files https://pla.esac.esa.int/#maps

Reply:
Yeah, so I'd just call this `SMICA`

Reply:
done